cdap-common/src/main/java/co/cask/cdap/common/AlreadyExistsException.java

```diff
@@ -32,11 +32,11 @@ public AlreadyExistsException(String errorMsg) {
     this.objectId = null;
   }
 
-  public AlreadyExistsException(EntityId entityId) {
+  public AlreadyExistsException(Object entityId) {
```

Comment:
rename the parameter, too, not just the type

Reply:
Is there any place in the code that assumes that it is always an entityId?

Reply:
No, the type of the field in this class is already an Object and the `getObjectId` method is not used